Sources/Network.swift

```diff
@@ -128,12 +128,12 @@ class Network {
                         parse: parse)
     }
 
-    class func trackIntegration(apiToken: String, serverURL: String, completion: @escaping (Bool) -> Void) {
-        let requestData = JSONHandler.encodeAPIData([["event": "Integration",
-                                                      "properties": ["token": "85053bf24bba75239b16a601d9387e17",
+    class func trackEvent(eventName: String, apiToken: String, distinctId: String, completion: @escaping (Bool) -> Void) {
```

Comment:
better rename it or put some comments to avoid people using it by mistake.

Reply:
might not be an issue for SPM but could be an issue if they include the source code.